OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -185,7 +202,6 @@ public virtual void PlayLaunchSounds()
 			var isAllied = Self.Owner.IsAlliedWith(renderPlayer);
 			Game.Sound.Play(SoundType.UI, isAllied ? Info.LaunchSound : Info.IncomingSound);
 
-			// IsAlliedWith returns true if renderPlayer is null, so we are safe here.
```

Comment:
Why are you changing this?

Reply:
I moved it to an XML comment, so it doesn't rot when the logic actually changes.